integrationTests/multiShard/block/executingMiniblocksSc_test.go

```diff
@@ -34,6 +34,7 @@ func TestProcessWithScTxsTopUpAndWithdrawOnlyProposers(t *testing.T) {
 		t.Skip("this is not a short test")
 	}
 
+	mingGasPrice := integrationTests.MinTxGasPrice
```

Comment:
add the minGasPrice on the testProcessorNode not as a global variable

Reply:
done